### ISSUE TITLE
"There was an issue distributing the post" - few other clues

### ISSUE THREAD

**Issue by pbaddock** [10up/distributor#616]
**Describe the bug**
I get a simple "There was an issue distributing the post" - with little additional info provided.
Tried enabling debugging and only clues (not sure if this is the prob) is this in debug log:
[18-Jul-2020 04:45:54 UTC] PHP Notice:  Object of class WP_Error could not be converted to int in /home/customer/www/foundation.makandulu.or.id/public_html/wp-content/plugins/distributor-stable/includes/push-ui.php on line 307
Using remote connection (not multisite) - have app passwords setup.  Connection showing green light / ok.

**Steps to Reproduce**

Go to master site, create new post.
Click on distributor link in wp admin top menu bar.
Select remote site connection from the list.
Click on distribute - get his message only "There was an issue distributing the post"

**Expected behavior**
Post is distributed

**Comment by pbaddock** [comment]
Traced back to wordfence issues.  I'm going to close this off for now and resolve whatever is blocking things from wordfence, tks for listening! lol.
will post again if I hit new issues. tks.

**Comment by jeffpaul** [comment]
@pbaddock could you provide more info on what you identified as conflicting with WordFence?  I'm curious if there's a way for us to identify that or at least call that out as a potential source issue in our documentation... thanks!

**Comment by pbaddock** [comment]
I had setup the sites on ServerA, and then moved them over to ServerB as part of some re-organisation of the staging environments I had seutp.  On ServerB I was simply getting that message "There was an issue distributing the post" without further details.

Eventually I disabled all wordfence setups on all sites and it started working.
I then re-enabled wordfence on the remote site (plan was to enable one by one to trace the source of the issue) but when I re-enabled again, things "just worked" as they were supposed to the first time.

So I am guessing perhaps some wordfence config needed to be updated or something on my site or were affected by the move, but after re-enabling it seems everything was ok.   I had updated .htaccess also with the remote app authorisation code snippet so perhaps wordfence hadn't picked up on that.   I'm sorry I can't be more specific helpful - to be honest I never really specifically identified the source, but instead found it started working above the above.
